projects/fourier/lib/fourier/utilities/system.rb

```diff
@@ -9,17 +9,17 @@ def self.system(*args)
         Kernel.system(*args) || exit(1)
       end
 
-      def self.tuist(*args, **kwargs)
-        @tuist = "/usr/local/bin/tuist"
+      def self.tuist(*args, source: false)
+        tuist = "/usr/local/bin/tuist"
 
-        if kwargs[:from_source] || !File.exist?(@tuist)
+        if source || !File.exist?(tuist)
```

Comment:
If you set `source` explicitly to `false` (i.e. `./fourier generate tuist --source=false`) this will still build from source if system tuist is not found, rather than raising an error.

Reply:
I don't think it's a big problem, but I also think defaulting to using source if system is not found is not super useful and could probably become confusing (as in this case), so I probably would avoid it